README.md

```diff
@@ -87,6 +89,33 @@
     helm install external-dns-unifi external-dns/external-dns -f external-dns-unifi-values.yaml --version 1.14.3 -n external-dns
     ```
 
+## Configuration
+
+### Unifi Controller Configuration
+
+| Environment Variable    | Description                                                  | Default Value |
+|-------------------------|--------------------------------------------------------------|---------------|
+| `UNIFI_USER`            | Username for the Unifi Controller (must be provided).        | N/A           |
+| `UNIFI_SKIP_TLS_VERIFY` | Whether to skip TLS verification (true or false).            | `true`        |
+| `UNIFI_SITE`            | Unifi Site Identifier (used in multi-site installations)     | `default`     |
+| `UNIFI_PASS`            | Password for the Unifi Controller (must be provided).        | N/A           |
+| `UNIFI_HOST`            | Host of the Unifi Controller (must be provided).             | N/A           |
+| `CONTROLLER_TYPE`       | Type of Unifi controller (e.g., gateway or standalone).      | `gateway`     |
```

Comment:
We should name this env var `UNIFI_CONTROLLER_TYPE` to fit in with the rest of the unifi env vars